garak/resources/tap/tap_main.py

```diff
@@ -233,6 +241,23 @@ def create_conv(self, full_prompt, system_prompt=None):
 
         conv = get_template(self.evaluation_generator.name)
         conv.set_system_message(system_prompt)
+        # Avoid sending overly long prompts.
+        # Crude and fast heuristic -- 100 tokens is about 75 words
+        if len(full_prompt.split()) / .75 > self.evaluator_token_limit:
+            # More expensive check yielding actual information -- add 100 token buffer to prompt
+            judge_system_prompt_tokens = token_count(system_prompt, self.evaluation_generator.name)
+            prompt_tokens = 100 + token_count(full_prompt, self.evaluation_generator.name)
+            # Iteratively reduce the prompt length
+            while judge_system_prompt_tokens + prompt_tokens >= self.evaluator_token_limit:
+                excess_tokens = judge_system_prompt_tokens + prompt_tokens - self.evaluator_token_limit
+                if excess_tokens > 0:
+                    # Truncate excess text
+                    excess_words = int(excess_tokens / .75)
```

Comment:
where's the .75 from? is it worth constant-ising?

Reply:
From line 246: `# Crude and fast heuristic -- 100 tokens is about 75 words`